cms/static/js/views/overview_assignment_grader.js

```diff
@@ -13,7 +13,7 @@ define(["js/views/baseview", "underscore", "gettext", "js/models/assignment_grad
                     // TODO move to a template file
                     '<h4 class="status-label"><%= assignmentType %></h4>' +
                     '<a data-tooltip="Mark/unmark this subsection as graded" class="menu-toggle" href="#">' +
-                        '<% if (!hideSymbol) {%><i class="icon fa fa-check"></i><%};%>' +
+                        '<% if (!hideSymbol) {%><span class="icon fa fa-check" aria-hidden="true"></span><%};%>' +
```

Comment:
This file appears to be dead code (I searched in edx-platform and found no references to it). Please delete it.